iothub/service/samples/PnpServiceSamples/noComponentSample/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
+
+namespace PnpServiceSample
+{
+    public class Program
+    {        
+        // Get connection string and device id inputs
+        private static readonly string hubConnectionString = Environment.GetEnvironmentVariable("IOTHUB_CONNECTION_STRING");
+        private static readonly string deviceId = Environment.GetEnvironmentVariable("DEVICE_ID");
+
+        // These are default values as per PnP no Component device client sample.
+        private const string propertyName = "targetTemperature";
+        private const double propertyValue = 60;
+        private const string methodToInvoke = "reboot";
```

Comment:
since these are not reused elsewhere, let us define it under the method block itself. That way you won't need to scroll up, and can get all the info at a single glance.

Reply:
Put all these on top for readability and when user wants to update for lets' say a new command, don't have to go to method to find these.